attractions/file-input/file-dropzone.svelte

```diff
@@ -28,17 +28,20 @@
   }
 
   async function acceptUpload(e) {
-    for (let file of (e.dataTransfer || e.target).files) {
+    const incomingFiles = Array.from((e.dataTransfer || e.target).files);
+    await Promise.all(incomingFiles.map(async (file) => {
```

Comment:
`Promise.allSettled` would allow us to remove the try-catch altogether, but its support is much more limited than `Promise.all`, so I'm just noting that we can consider it if we will add Babel or something later on